apps/wallet/src/ui/app/experimentation/feature-gating.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { GrowthBook } from '@growthbook/growthbook';
+
+export default class FeatureGating {
+    private _growthBook: GrowthBook;
+    constructor() {
+        // Create a GrowthBook context
+        this._growthBook = new GrowthBook();
+    }
+
+    public async init() {
+        if (process.env.NODE_ENV !== 'production') {
```

Comment:
Rather than this, can we just set up an environment in growthbook for development?

Reply:
done!